core/tests/protractor/accessibility.js

```diff
@@ -58,8 +59,11 @@ describe('screenreader and keyboard user accessibility features', function() {
   var DONATE_URL = 'http://localhost:9001/donate';
 
   var holdCtrlAndPressKey = async function(key) {
-    await browser.actions().sendKeys(
-      protractor.Key.chord(protractor.Key.CONTROL, key)).perform();
+    await action.sendKeys(
+      'Browser actions',
+      browser.actions(),
+      protractor.Key.chord(protractor.Key.CONTROL, key)
+    ).perform();
```

Comment:
This change isn't required. `browser.actions()` isn't an element, it creates an object we can call `sendKeys` on. The [Selenium documentation](https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/interactions/Actions.html#sendKeys(java.lang.CharSequence...)) describes this. Therefore, it's okay to leave `browser.actions().sendKeys(...)` statements throughout this file

Reply:
Done